modules/installation/pages/general-requirements.adoc

```diff
@@ -25,19 +25,18 @@ It is also not possible to use {slsa} for SAP as a base for {productname} for a
 [[install.scc-register]]
 == Obtaining your SUSE Customer Center Credentials
 
-You will need to create an account with {scc} before you install {sls} and {productname}.
-To obtain your {scc} credentials:
+Create an account with {scc} before installation of {sls} and {productname}.
 
 [[creating.scc.account.mgr]]
 .Procedure: Obtaining Your SCC Organization Credentials
 . Navigate to https://scc.suse.com/login in your Web browser.
 . Log in to your SCC account, or follow the prompts to create a new account.
```

Comment:
In the following, these are not buttons but other GUI elements.  Thus I'd rather vote to use [guimenu] (or menu: if appropriate).